recipes-connectivity/bluez5/bluez5/0002-QCA9377-set-bt-mac-from-hciattach-cmd.patch

```diff
@@ -0,0 +1,61 @@
+Index: bluez-5.49/tools/hciattach_rome.c
+===================================================================
+--- bluez-5.49.orig/tools/hciattach_rome.c
++++ bluez-5.49/tools/hciattach_rome.c
+@@ -29,7 +29,7 @@
+  *
+  ******************************************************************************/
+ 
+-#define MODULE_HAS_MAC_ADDR
++//#define MODULE_HAS_MAC_ADDR
```

Comment:
This is not ok as it prevents the usage of the fused mac address in the module which we still want as default behavior.

Reply:
Thank you for the comment. However this just adds another level of control in addition to the fused mac address in the module.

Reply:
I think it's my bad to commented out '#define MODULE_HAS_MAC_ADDR'. I should remove this if you agree on the idea of add BT mac address from hciattach command input

Reply:
Yes I'm ok with the idea but need to keep the default behavior the same.

Reply:
> This is not ok as it prevents the usage of the fused mac address in the module which we still want as default behavior.

I agreed and will close this PR. Thank you for the review.